server/models/Message.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const messageSchema = new Schema({
+  content: { type: String },
+  author: { type: Schema.Types.ObjectId, ref: "User" },
+  conversation: { type: Schema.Types.ObjectId, ref: "Conversation" },
```

Comment:
can we make this indexed (look into MongoDB indexes and indexes in Mongoose)

Reply:
Okay I indexed the conversation refs, unsure If you wanted the author as well?